assets/dev/scss/frontend/_container.scss

```diff
@@ -50,42 +44,62 @@
 	overflow: var( --overflow );
 	transition: background var(--background-transition, 0.3s), border var(--border-transition, 0.3s), box-shadow var(--border-transition, 0.3s), transform var(--e-con-transform-transition-duration, 0.4s);
 
+	&.e-flex {
```

Comment:
I would recommend creating separate files to avoid messy code. 

/frontent/container/_base.scss
/frontent/container/_flex.scss
/frontent/container/_grid.scss

Reply:
But we can keep it in one file as well. 
Your decision.